app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from .errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
```

Comment:
According to the checklist, you should use absolute imports instead of relative imports. Change `from .errors import` to `from app.errors import`.